ansible_base/resource_registry/tasks/sync.py

```
@@ -223,12 +224,17 @@ def _paginate(self, list_fn, actor_id_key: str, assignment_type: str) -> bool:
 
         Returns True if all pages were fetched successfully, False on any error.
         """
-        page = 1
+        next = None
```

Ruff (0.15.15)

[error] Variable `next` is shadowing a Python builtin

(A001)